### PR TITLE
Azure Communication Services Email (Certified Connector)

### PR DESCRIPTION
**Pull request by lei-msft**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by lei-msft** [comment]
For email, they do not specify individual error codes ([Email - Send - REST API (Azure Communication) | Microsoft Docs](https://docs.microsoft.com/en-us/rest/api/communication/email/send), [Email - Get Send Status - REST API (Azure Communication) | Microsoft Docs](https://docs.microsoft.com/en-us/rest/api/communication/email/get-send-status)) and instead list all non-200 as errors using the "default" flag. I copied over the same to the connector in this PR

File: certified-connectors/Azure Communication Services Email/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+            "headers": {
+              "x-ms-error-code": {
+                "description": "Error code - this will be the same as the code in the error property in the response body.",
+                "type": "string"
+              }
+            }
+          },
+          "202": {
+            "description": "Accepted.",
+            "schema": {},
```
Comment (v-EgorKozhadei) [inline certified-connectors/Azure Communication Services Email/apiDefinition.swagger.json:187]: @lei-msft empty response schemas are [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).

File: certified-connectors/Azure Communication Services Email/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+### Get Email Message Status
+
+Gets the status of an email message sent previously.
+
+## Obtaining Credentials
+
+You can create a new connection using an [Azure Communication Services resource connection string](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp#access-your-connection-strings-and-service-endpoints).
+
+## Deployment Instructions
```
Comment (v-EgorKozhadei) [inline certified-connectors/Azure Communication Services Email/README.md:42]: @lei-msft please review the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). The headings should follow the order from the template, all the required headings should be present.

File: certified-connectors/Azure Communication Services Email/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+          "tooltip": "Find your connection string in your Azure Communication Services resource in Azure Portal in the \"Keys\" tab.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Azure Communication Services Email/apiProperties.json:18]: @lei-msft this color is a default one and it's [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector) to use it. Can you please tell me what is the color of your icon and how will it look like? Is the icon Azure-related?

**Comment by v-EgorKozhadei** [comment]
Hello @lei-msft, 

I hope you are doing well. 

I am following up with you to see how things are going with preparing your connector. 

Please let us know if you have any questions. 

Thank you.

**Review by v-EgorKozhadei — APPROVED** [review]

**Pull request merged**